archive/Thematic/2021/March/progressed/mysterium3/empty.html

```diff
@@ -0,0 +1,28 @@
+
+<h1 class="mc-toc-title" dir="ltr" style="text-align: center;display: block;margin: 0;padding: 0;color: #111111;font-family: 'Merriweather Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;font-size: 26px;font-style: normal;font-weight: bold;line-height: 125%;letter-spacing: normal;"><span style="font-family:trebuchet ms,lucida grande,lucida sans unicode,lucida sans,tahoma,sans-serif"><span style="font-size:30px">Crypto and Passive Income - What’s the Deal?</span></span></h1>
```

Comment:
Usually i'm moving information into content.md and empty.html
Not because I want to do it. But there can be some changes in content - and I'm tracking it.

not a problem at all - but if the client decides to change something on the fly(and they always doing it) - it will force us to make another cycle of changes.

Just to let you know